test/unit/shady-content.html

```diff
@@ -12,10 +12,9 @@
 <head>
   <meta charset="utf-8">
   <script>
-    if (!window.customElements) {
-      window.customElements = {};
+    if (window.customElements) {
+      customElements.forcePolyfill = true;
```

Comment:
this change seems unrelated to the intent of the CL?

Reply:
Separated out to https://github.com/Polymer/polymer/pull/5268

Reply:
thank you